.mailmap

```diff
@@ -75,12 +84,17 @@ Matej Košík <matej.kosik@inria.fr>                 Matej Košík <mail@matej-k
 Ambroise Lafont <chaster_killer@hotmail.fr>        amblaf <you@example.com>
 Ambroise Lafont <chaster_killer@hotmail.fr>        Ambroise <chaster_killer@hotmail.fr>
 Vincent Laporte <Vincent.Laporte@fondation-inria.fr> Vincent Laporte <Vincent.Laporte@gmail.com>
+Vincent Laporte <Vincent.Laporte@fondation-inria.fr> Vincent Laporte <Vincent.Laporte@inria.fr>
```

Comment:
I think the Fondation Inria one was the former, and the Inria one is the newer.

Reply:
Updated